plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeRedirectionsProvider.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake;
+
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.connector.TableScanRedirectApplicationResult;
+
+import java.util.Optional;
+
+public interface DeltaLakeRedirectionsProvider
+{
+    DeltaLakeRedirectionsProvider NOOP = (session, tableHandle) -> Optional.empty();
```

Comment:
nit: rename to NO_REDIRECTIONS and static import

Reply:
NO_REDIRECTIONS could be used for table scan redirections .. let me keep as is

Reply:
it;s used in single place only anyway